config/manager/manager.yaml

```diff
@@ -13,9 +13,13 @@ spec:
       labels:
         control-plane: controller-manager
     spec:
+      securityContext:
+        runAsUser: 1001
```

Comment:
For us, there's no reason why we couldn't use
```
USER 65532:65532
```
(and this security context) as suggested by the guide, since we don't mount any volumes. Openshift will override the user and group anyway.